### PR TITLE
NWPS-124 - Newsletter

### PR DESCRIPTION
**Pull request by Raq-HEE**
Hi @sankar-shunmuga / @adaMiclos - this is for the backend piece only. The JS validation is being PR'd seperately.

### CODE REVIEW

File: site/components/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml [hunk 1]
```diff
@@ -45,6 +45,11 @@
         <entry key="newsCategories_medical" value="/content/documents/administration/valuelists/medical/newscategories"/>
         <entry key="newsCategories_website-testing" value="/content/documents/administration/valuelists/website-testing/newscategories"/>
 
+        <!-- Newsletter Subscribe Form Value Lists -->
+        <entry key="newsletterRegions" value="/content/documents/administration/valuelists/newsletterregions"/>
```
Comment (sankar-shunmuga) [inline site/components/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml:49]: As discussed, please create newsletterRegions valuelists per channel

File: site/components/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml [hunk 2]
```diff
@@ -45,6 +45,11 @@
...
+        <entry key="newsletterOrganisations" value="/content/documents/administration/valuelists/newsletterorganisations"/>
```
Comment (sankar-shunmuga) [inline site/components/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml:50]: As discussed, please create newsletterOrganisations valuelists per channel

File: site/components/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml [hunk 3]
```diff
@@ -45,6 +45,11 @@
...
+        <entry key="newsletterProfessions" value="/content/documents/administration/valuelists/newsletterprofessions"/>
```
Comment (sankar-shunmuga) [inline site/components/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml:51]: As discussed, please create newsletterProfessions valuelists per channel

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/newsletterorganisations.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
+/content/documents/administration/valuelists/newsletterorganisations:
```
Comment (sankar-shunmuga) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/newsletterorganisations.yaml:1]: As discussed, please create valuelists per channel

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/newsletterprofessions.yaml [hunk 5]
```diff
@@ -0,0 +1,817 @@
+/content/documents/administration/valuelists/newsletterprofessions:
```
Comment (sankar-shunmuga) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/newsletterprofessions.yaml:1]: As discussed, please create valuelists per channel

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/newsletterregions.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
+/content/documents/administration/valuelists/newsletterregions:
```
Comment (sankar-shunmuga) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/newsletterregions.yaml:1]: As discussed, please create valuelists per channel

**Review by sankar-shunmuga — CHANGES_REQUESTED** [review]
Hi @Raq-HEE , As discussed, please create valuelists per channel

**Comment by Raq-HEE** [comment]
Thatnks for the review San, discussed with team and wer're going for global valuelists for now.

**Pull request merged**